specification/dfp/resource-manager/readme.go.md

```diff
@@ -0,0 +1,26 @@
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+```yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  namespace: dfp
+  clear-output-folder: true
+```
+
+### Go multi-api
+
+``` yaml $(go) && $(multiapi)
+batch:
+  - tag: package-2021-02-01-privatepreview-preview
+```
+
+### Tag: package-2021-02-01-privatepreview-preview and go
+
+These settings apply only when `--tag=package-2021-02-01-privatepreview-preview --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+```yaml $(tag) == 'package-2021-02-01-privatepreview-preview' && $(go)
+output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2021-02-01-privatepreview/$(namespace)
```

Comment:
simply preview is fine here - we will not release anything in private preview anyway.
```suggestion
output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2021-02-01-preview/$(namespace)
```

Reply:
Fixed